tests/python/pants_test/pantsd/test_process_manager.py

```diff
@@ -6,13 +6,15 @@
                         unicode_literals, with_statement)
 
 import errno
+import logging
 import os
 import sys
 from contextlib import contextmanager
 
 import mock
 import psutil
 
+from pants.pantsd import process_manager
```

Comment:
this new import is unused/redundant, afaict

Reply:
Ah, yup - thanks.